modules/k8s/pod.go

```diff
@@ -148,3 +148,19 @@ func IsPodAvailable(pod *corev1.Pod) bool {
 	}
 	return pod.Status.Phase == corev1.PodRunning
 }
+
+// GetPodLogsE returns the logs of a Pod at the time when the function was called. If the Pod is not running an Error is returned.
+func GetPodLogsE(t testing.TestingT, options *KubectlOptions, pod *corev1.Pod) (string, error) {
+	output, err := RunKubectlAndGetOutputE(t, options, "logs", pod.Name)
```

Comment:
May be helpful to fetch logs from specific container inside of pod, by passing "-c <name>" option

Reply:
I have added the option to pass a containerName. An empty string will just give you the logs of a single container pod. 
Both branches are tested.